src/main/java/view/InputView.java

```diff
@@ -0,0 +1,45 @@
+package view;
+import camp.nextstep.edu.missionutils.Console;
+import view.constants.SystemMessage;
+
+public class InputView {
+    public String inputPurchaseAmount() {
+        while (true) {
+            try {
+                System.out.println(SystemMessage.INPUT_PURCHASE_AMOUNT.getSystemMessage());
+                String input = Console.readLine();
+
+                return input;
+
+            } catch(IllegalArgumentException e) {
+                System.out.println("[ERROR] " + e.getMessage());
+            }
+        }
+    }
+
+    public String inputWinningNumbers() {
+        while (true) {
+            try {
+                System.out.println(SystemMessage.INPUT_WINNING_NUMBERS.getSystemMessage());
+                String input = Console.readLine();
+
+                return input;
+            } catch (IllegalArgumentException e) {
+                System.out.println("[ERROR] " + e.getMessage());
+            }
+        }
+    }
+
+    public String inputBonusNumber() {
+        while (true) {
+            try {
+                System.out.println(SystemMessage.INPUT_BONUS_NUMBER.getSystemMessage());
+                String input = Console.readLine();
+                return input;
+            } catch (IllegalArgumentException e) {
+                System.out.println("[ERROR] " + e.getMessage());
+            }
+        }
+    }
```

Comment:
```suggestion
    public String inputBonusNumber() {
        System.out.println(SystemMessage.INPUT_BONUS_NUMBER.getSystemMessage());
        return Console.readLine();
    }
```

input에서는 단순히 콘솔에서 값만 받아오기 때문에 try-catch로 잡는 로직이 필요없을 것 같습니다